$MirOS: ports/mailnews/pine/patches/patch-imap_src_ipopd_ipopd_8,v 1.1.7.1 2005/03/18 15:49:48 tg Exp $
$OpenBSD: patch-ipopd.8c,v 1.4 2003/10/05 22:00:00 jakob Exp $
--- imap/src/ipopd/ipopd.8.orig	Tue May 18 17:49:43 2004
+++ imap/src/ipopd/ipopd.8	Wed May  4 23:28:07 2005
@@ -3,9 +3,9 @@
 .SH NAME
 IPOPd \- Post Office Protocol server
 .SH SYNOPSIS
-.B /usr/etc/ipop2d
+.B ipop2d
 .PP
-.B /usr/etc/ipop3d
+.B ipop3d
 .SH DESCRIPTION
 .I ipop2d
 and
